Code/Framework/AzCore/Tests/EBus.cpp

```diff
@@ -4180,6 +4180,124 @@ namespace UnitTest
         EXPECT_EQ(totalThreadDispatchCalls, ThreadDispatchTestBusTraits::s_threadPostDispatchCalls);
         ThreadDispatchTestBusTraits::s_threadPostDispatchCalls = 0;
     }
+
+
+    struct ReentrantEbusUseTestRequests : public AZ::EBusTraits
```

Comment:
The name of the struct should use `EBus` with a capital `B` instead of lowercase B to match the EBus naming scheme
```suggestion
    struct ReentrantEBusUseTestRequests : public AZ::EBusTraits
```

Reply:
Fixed!